panel/models/perspective.ts

```diff
@@ -166,7 +166,7 @@ export class PerspectiveView extends HTMLBoxView {
     const container = div({
       class: "pnx-perspective-viewer",
       style: {
-        zIndex: 0,
+        zIndex: "0",
```

Comment:
```ts
    panel/models/perspective.ts:166:27 - error TS2345: Argument of type '{ class: string; style: { zIndex: number; }; }' is not assignable to parameter of type 'HTMLChild | HTMLAttrs<"div", {}>'.
      Types of property 'style' are incompatible.
        Type '{ zIndex: number; }' is not assignable to type 'CSSStyles'.
          Type '{ zIndex: number; }' is not assignable to type 'CSSStylesCamel'.
            Types of property 'zIndex' are incompatible.
```
The correct type is `string`, but some CSS APIs allow `any` and convert to `string`. I allow only strings to make sure the conversion is intentional and not a bug (`any` will gladly accept anything without second thoughts and CSS parser/validator doesn't issue any errors or even warnings). There aren't many CSS properties that allow plain numbers (e.g. `width: 100;` is invalid), but I can make an exception for `z-index` and perhaps some more.

Reply:
There's no need for that; I like the strictness. 

I just didn't have time to look at it Friday, and just pasted in the build errors in the first post.